runtime/doc/develop.txt

```diff
@@ -434,7 +434,9 @@ Use existing common {verb} names (actions) if possible:
     - eval:         Evaluates an expression
     - exec:         Executes code, may return a result
     - fmt:          Formats
-    - get:          Gets things (often by a query)
+    - get:          Gets things. Two variants (overloads):
+                    1. `get(id: int): any` returns one item.
+                    2. `get(filter: dict): any[]` returns a list.
```

Comment:
Would it be an overkill to use a generic notation here to indicate that type specificity should be used (when possible)?

```suggestion
                    1. `get<T>(id: int): T` returns one item.
                    2. `get<T>(filter: dict): T[]` returns a list.
```